firmware/src/main.cpp

```diff
@@ -185,8 +185,8 @@ int generic_handler(const char *path, const char *types, lo_arg ** argv,
 
 struct Lm {
     mpr_sig fsr = 0;
-    float fsrMax = 4900;
-    float fsrMin = 0;
+    int fsrMax = 2000;
+    int fsrMin = 0;
```

Comment:
Did you check that in practice? I think the max value is 4096 as the FSR offset mapping/range works as :
- offset (min) to 4096 (max) -> 0 (min) to 4096

It doesn't (or shouldn't) change the range but remove the offset of the residual reading from the shrink pressure and sets the same range for all cases. That ensures we have the same range regardless of the offset. 

in that case, the min and max should be as my suggestion, and we need to check if the FSR offset is working as intended.

```suggestion
    int fsrMax = 4096;
    int fsrMin = 0;
```